transports/sql/index.md

```diff
@@ -20,16 +20,17 @@ The SQL Server transport implements a message queuing mechanism on top of [Micro
 
 ## Transport at a glance
 
-|Feature                    |   |
-|:---                       |---
-|Transactions |None, ReceiveOnly, SendsAtomicWithReceive, TransactionScope
-|Pub/Sub                    |Native
-|Timeouts                   |Native
-|Large message bodies       |SqlServer can handle arbitrary message size within available resources, very large messages via data bus
-|Scale-out             |Competing consumer
-|Scripted Deployment        |Sql Scripts
-|Installers                 |Optional
-|Native integration         |[Supported](native-integration.md)
+| Feature                                            |                                                                                                          |
+|:---------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| Transactions                                       | None, ReceiveOnly, SendsAtomicWithReceive, TransactionScope                                              |
+| Pub/Sub                                            | Native                                                                                                   |
+| Timeouts                                           | Native                                                                                                   |
+| Large message bodies                               | SqlServer can handle arbitrary message size within available resources, very large messages via data bus |
+| Scale-out                                          | Competing consumer                                                                                       |
+| Scripted Deployment                                | Sql Scripts                                                                                              |
+| Installers                                         | Optional                                                                                                 |
+| Native integration                                 | [Supported](native-integration.md)                                                                       |
+| [time-to-be-received](#time-to-be-received) (TTBR) | Storage reclaimed at most 5 minutes after expiration when receiving endpoint is running                  |
```

Comment:
Why 5 minutes? SQLT has a (disabled by default) feature to remove the expired messages on startup but it runs once (when the endpoint starts), not periodically.

Reply:
@SzymonPobiega ☝️